CHANGES.rst

```diff
@@ -8,11 +8,26 @@ Unreleased
 
 * ``config_interpolation`` parameter for :class:`.Reddit` supporting basic and
   extended modes.
+* Methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` contains a new parameter,
+  ``fetch``, that toggles the automatic fetching of existing data from Reddit.
+  It is set to true on default.
+* Values in methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` that are left as the
+  defaults will no longer be over-written if the ``fetch`` parameter is set to
+  true, but will fill in existing values for the flair template.
+* The parameter ``text`` for methods
+  :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` is no longer required.
 
 **Removed**
 
 * Converting :class:`.APIException` to string will no longer escape unicode
   characters.
+* Methods :meth:`.SubredditLinkFlairTemplates.update` and
+  :meth:`.SubredditRedditorFlairTemplates.update` will no longer
+  create flairs that are using an invalid template id, but instead throw a
+  :class:`.InvalidFlairTemplateID`.
```

Comment:
I'm having second thoughts about this. While it would be a mis-use, for current versions of PRAW it is indeed valid to use this method to create flair templates with new (currently invalid) IDs. This change, while good overall, would also break existing workflows.

Reply:
How about we add a DeprecationWarning that fixes this in the next major release. For that matter, I’ll subclass the exception as DeprecationWarning.

Reply:
Yes, it could perhaps be a `DeprecationWarning`. However, I see no need to subclass your exception from `DeprecationWarning`, as your exception should be a `ClientException` once it is introduced.

Reply:
It’ll be temporary, removed after the next major update.

Reply:
There's no need to subclass an exception and then only throw the subclass in one place. Further, a `DeprecationWarning` is most appropriate since we would be warning about deprecation, rather than actually complaining about the nonexistent ID.

Reply:
Ahh. So what should I do with the exception?

Reply:
Just throw a `DeprecationWarning`. You can leave the `InvalidFlairTemplateID` in (but not raise it anywhere).